src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,130 @@
-export const PeopleTable = () => {
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+interface Props {
+  people: Person[]
+  link: string
+  sort: string
+  order: string
+}
+
+export const PeaopleTable: FC<Props> = ({
```

Comment:
There is a typo in the component name, it should be 'PeopleTable' instead of 'PeaopleTable'.